exercises/practice/bottle-song/.meta/example.ex

```diff
@@ -0,0 +1,47 @@
+defmodule BottleSong do
```

Comment:
Maybe a quick moduledoc would be nice?
You can adapt the first line of the instructions